services/explorer/graphql/server/graph/schema/types.graphql

```diff
@@ -225,3 +224,21 @@ enum KappaStatus{
   PENDING
   UNKNOWN
 }
+
+input ContractQuery {
```

Comment:
if one of these is required (I think both may be?) should add a `!` (note: coordinate w/ @bigboydiamonds I thikn this may require a fe change)

Reply:
#1482